src/TableRecordInstantiator.php

```diff
@@ -38,12 +38,6 @@ public function instantiate(Fixture $fixture)
         $tablename = substr($fixture->getClass(), 6);
         $r = new TableRecord($tablename);
 
-        if (!isset($this->ids[$tablename])) {
```

Comment:
We can use flag, but, flag applied for all tables and this not solve problem because raised another problem.
I think better solution (as we still using this at old code) is to mark v2.0.0 tags.